app/src/ui/history/compare.tsx

```diff
@@ -137,7 +137,7 @@ export class CompareSidebar extends React.Component<
             placeholder={placeholderText}
             onFocus={this.onTextBoxFocused}
             value={filterText}
-            disabled={allBranches.length <= 1}
+            disabled={allBranches.length < 1}
```

Comment:
We can just check if it's equal to 0 here and 👇

Reply:
I realize now that it doesn't show the entire diff, but I was pointed at the change that follows this one.